src/express.re

```diff
@@ -115,10 +115,18 @@ module Middleware = {
 module MakeBindFunctions (T: {type t;}) => {
   external use : T.t => Middleware.t => unit = "" [@@bs.send];
   external useN : T.t => array Middleware.t => unit = "use" [@@bs.send];
-  external useOnPath : T.t => path::string => Middleware.t => unit = "use" [@@bs.send];
-  external useOnPathN: T.t => path::string => array Middleware.t => unit = "use" [@@bs.send];
-  external get : T.t => path::string => Middleware.t => unit = "" [@@bs.send];
-  external getN: T.t => path::string => array Middleware.t => unit = "" [@@bs.send];
+  external useOnPath : T.t => path::string => Middleware.t => unit 
+    = "use" [@@bs.send];
+  external useOnPathN: T.t => path::string => array Middleware.t => unit 
+    = "use" [@@bs.send];
+  external get : T.t => path::string => Middleware.t => unit 
+    = "" [@@bs.send];
+  external getN: T.t => path::string => array Middleware.t => unit 
+    = "get" [@@bs.send];
+  external post : T.t => path::string => Middleware.t => unit 
+    = "" [@@bs.send];
+  external postN: T.t => path::string => array Middleware.t => unit 
```

Comment:
I've used `*Many` in the `Js` API: https://bloomberg.github.io/bucklescript/api/Js_string.html#VALfromCharCodeMany

But something like `Middleware.sequence [|m1, m2|]` or `Middleware.(m1 |> then_ m2)` would be more descriptive.